server/src/lib/model/user-client.ts

```diff
@@ -63,41 +64,43 @@ const generateAccentToken = (accent: string) => {
   return accent
     .trim()
     .toLowerCase()
-    .replace(/[\.,;\?\!'"]/g, '')
-    .replace(/[\s-_\(\)\[\]\<\>`]/g, '-')
+    .replace(/[.,;?!'"]/g, '')
+    .replace(/[\s-_()[\]<>`]/g, '-')
```

Comment:
What was the thinking behind removing the escapes?

Reply:
So it the linter picked it up as https://eslint.org/docs/rules/no-useless-escape

Looking into it more it looks like:
+ anything inside of a regex set `[]` doesn't need to be escaped: https://stackoverflow.com/questions/10593387/when-do-i-need-to-escape-characters-within-a-regex-character-set-within only exception is the `]`
+ `<` and `>` doesn't need to be escaped

Can change it back as it makes no difference to code, just style at this point

Reply:
Huh, TIL! good to know, this is good to go. Thanks :)